packages/truffle/webpack.config.js

```diff
@@ -104,6 +104,7 @@ module.exports = {
     // Here, we leave it as an external, and use the original-require
     // module that's a dependency of Truffle instead.
     /^original-require$/,
+    /^ganache$/,
```

Comment:
`ganache` includes a couple of native dependencies that would need to be copied to the webpack destination if we were to bundle it with Truffle. Since that is more complicated, I've just excluded it from the bundling and added it to the runtime dependencies.

The most optimal build would be for ganache to ship an unpacked version that doesn't bundle anything at all, that way Truffle's webpack could tree-shake and dedupe.